crates/aptos/src/node/mod.rs

```diff
@@ -515,12 +556,12 @@ impl CliCommand<serde_json::Value> for ShowValidatorSet {
     }
 }
 
-async fn get_resource_migration(
+async fn get_resource_migration<T: DeserializeOwned>(
     client: &aptos_rest_client::Client,
     address: AccountAddress,
     original_resource: &'static str,
     new_resource: &'static str,
-) -> CliTypedResult<Response<serde_json::Value>> {
+) -> CliTypedResult<Response<T>> {
```

Comment:
This function should just be dropped now.  This was for Upper to Lower migration.